thirdeye/thirdeye-frontend/app/pods/components/rootcause-custom-baseline/component.js

```diff
@@ -26,10 +22,11 @@ export default Component.extend({
 
   selectedBaselineType: 'Unit over x units',
 
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
You can set a `BASELINE_TYPES` constant above the component declaration and reference it here so you don't have to bypass eslint.